src/Bowling.js

```diff
@@ -0,0 +1,43 @@
+function Bowling () {
+  this.rolls = []
+}
+Bowling.prototype.roll = function (pins) {
+  this.rolls.push(pins)
+}
+Bowling.prototype.score = function () {
+  var result = 0
+  var rollIndex = 0
+  var bowling = this
+  for (var frameIndex = 0; frameIndex < 10; frameIndex++) {
+    if (isStrike()) {
+      result += getStrikeScore()
+      rollIndex++
+    } else if (isSpare()) {
+      result += getSpareScore()
+      rollIndex += 2
+    } else {
+      result += getNormalScore()
+      rollIndex += 2
+    }
+  }
+  return result
```

Comment:
Is this return a part of a function? Is it part of the .score function? Not too sure on why it is here.